spec/factories.rb

```diff
@@ -1,2 +1,5 @@
 FactoryBot.define do
+  factory :budget do
+    sequence(:name) { |n| "name#{n}" }
```

Comment:
A suggestion to change to "budget#{n}" which will help with debugging what objects you work with in the specs